testCommon/src/testFixtures/resources/setup-for-overridden-fork-point.sh

```diff
@@ -25,7 +25,6 @@ cd machete-sandbox
   git branch -d root
 
   git config machete.overrideForkPoint.build-chain.to "$(git rev-parse allow-ownership-link)"
-  git config machete.overrideForkPoint.build-chain.whileDescendantOf "$(git rev-parse build-chain)"
```

Comment:
CLI 3.15.x would have a different output for this script (since it still requires `whileDescendantOf` to be present for override to apply), hence a regeneration was needed to keep stuff consistent